ca/config_test.go

```diff
@@ -264,17 +266,22 @@ func TestLoadSecurityConfigIntermediates(t *testing.T) {
 	rootCA, err := ca.NewRootCA(testutils.ECDSACertChain[2], nil, nil, ca.DefaultNodeCertExpiration, nil)
 	require.NoError(t, err)
 
+	ctx := log.WithLogger(context.Background(), log.L.WithFields(logrus.Fields{
```

Comment:
On all the other ones it was `tc.Context` (for "test CA"), that uses `cautil.go`'s `TestCA` object.  This particular test doesn't create one of those, hence creating the context manually.